### DISCUSSION TITLE
Error while trying to update the metadata using Nhentai.

### DISCUSSION THREAD

**Discussion by Deejay85** [Difegue/LANraragi#532] [Q&A]
I'm currently running the latest version of Lanraragi, and am running Windows 10, and there is a slight problem when trying to update the metadata using Nhentai. I run the program for a bit, but the message I get is as follows:

Plugin error while processing ID 
51a0bae1973f7d2d9d9d37629eeed1d6fb3d7486(Connect timeout at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/nHentai.pm line 98.

I looked the file up, as well as the line number, but it doesn't look like it's anything I can fix, and if I had to guess, it may be something I'm doing wrong, but since it worked on 8.0, I'm not sure what it could be. I did uninstall the last version before reinstalling this one, but I doubt that had anything to do with it.

**Comment by Difegue** [comment]
Connect timeout means the server can't connect to NH for some reason.

Do other metadata plugins work? (Chaika or E-H) 

I don't think NHentai has cloudflare anti-ddos or something like that, but you might want to load the website in your web browser just to check if you can reach it correctly.

**Comment by Deejay85** [comment]
I tried both Chaika and Ehentai, and I got the same result. I also uninstalled, reinstalled, and restarted the computer to make sure it wasn't something on my end, and so far no luck. Also, interestingly enough, I was able to log into Nhentai, so that's not the issue either.

2cdda0f9c587158f66eeb9508c353278d3c12dec(Connect timeout at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/Chaika.pm line 143.


2cdda0f9c587158f66eeb9508c353278d3c12dec(Connect timeout at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/EHentai.pm line 230.

**Comment by Difegue** [comment]
Looks like an issue with the distro being overall unable to reach external internet services then - You might have some luck with the WSL2 option, as that has a different network stack.
